frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -89,5 +79,15 @@ export function Onboarding(): JSX.Element | null {
         }
     }, [])
 
-    return product ? getOnboarding(product.type) : null
+    if (!product) {
+        return <></>
+    }
+    const onboardingViews = {
```

Comment:
Just commenting here to have somewhere to comment. It's not clear how I navigate back from the page to the previous area. Maybe a little back button would be useful?

<img width="935" alt="Screenshot 2023-09-06 at 11 31 53" src="https://github.com/PostHog/posthog/assets/2536520/0f381e7b-7009-41ca-90b3-4313e7017bb5">

Reply:
✅ 
![image](https://github.com/PostHog/posthog/assets/18598166/59b1dfe4-6c9f-4b12-a4ed-94f98af8bfc1)